semgrep-core/src/core/Metavariable.ml

```diff
@@ -88,6 +88,19 @@ let mvalue_to_any = function
   | T x -> G.T x
   | P x -> G.P x
 
+let program_of_mvalue : mvalue -> G.program option = function
```

Comment:
I would add a comment here "This is used for metavariable-pattern: where we need to transform the content of a metavariable in a program so we can use evaluate_formula on it".